.github/workflows/tests.yml

```
@@ -13,10 +13,13 @@ on:
       - released
 
 jobs:
-  build:
+  tests:
+    runs-on: ubuntu-20.04
```

We should bump this forward to the latest LTS release. Our EC2 development box is on Ubuntu 22.04

```suggestion
    runs-on: ubuntu-20.04
```